BARJ_NERDS.ino

```diff
@@ -0,0 +1,26 @@
+// Driver Assistive System (DAS)
+// 2022/09
+// Team 3 in UC Davis Transfer Bridge
+// Phase 1:
+//   Implement the headlight system of DAS
+//   DHT11 (Calculate temperature dewpoint spread from temperature and humidity)
```

Comment:
Why? I'm not sure what the dewpoint has to do with headbeam brightness calculations?